tests/Unit/NumericalAlgorithms/Interpolation/CMakeLists.txt

```diff
@@ -29,7 +30,7 @@ add_test_library(
   ${LIBRARY}
   "NumericalAlgorithms/Interpolation/"
   "${LIBRARY_SOURCES}"
-  "Interpolation;MathFunctions"
+  "Interpolation;MathFunctions;Utilities"
```

Comment:
I'm confused. What part of `Utilities` needs to get linked in? I don't see anything in the code you added that includes anything from `Utilities`